components/InterviewCard.tsx

```
@@ -0,0 +1,104 @@
+import dayjs from "dayjs";
+import Link from "next/link";
+import Image from "next/image";
+
+import { Button } from "./ui/button";
+import DisplayTechIcons from "./DisplayTechIcons";
+
+import { cn, getRandomInterviewCover } from "@/lib/utils";
+
+const InterviewCard = async ({
+  interviewId,
+  userId,
```

ESLint

[error] 'userId' is defined but never used.

(@typescript-eslint/no-unused-vars)